post_installation_configuration/preparing-for-users.adoc

```diff
@@ -100,6 +100,10 @@ include::modules/rbac-viewing-local-roles.adoc[leveloffset=+2]
 
 include::modules/rbac-adding-roles.adoc[leveloffset=+2]
 
+include::modules/unauthenticated-users-cluster-role-binding-con.adoc[leveloffset=+1]
```

Comment:
I don't think that these are added at the right level. Here's what it looked like prior to your addition:

![image](https://github.com/openshift/openshift-docs/assets/7443887/b78af4f7-8b08-4425-b86a-7c9cd353492a)

And after: 

![image](https://github.com/openshift/openshift-docs/assets/7443887/9d523061-f9c9-4e20-8a92-74926f7e1fd0)

So now it looks like "Creating a local role" is nested under your adding unauthenticated groups procedure, which I don't believe was your intention. 

I'm guessing you just need to bump these offsets to be +2, but just make sure that it looks like how you expect it to and where you want it to in the TOC